inc/Utils.php

```diff
@@ -59,6 +59,9 @@ public static function get_supported_mime_types(): array {
 		$supported_types = array_values( get_allowed_mime_types() );
 		$allowed_types   = array_intersect( $allowed_types, $supported_types );
 
+		// WP uses jpeg for all jpeg types, so we need to add jpg and jpe as well explicitly.
+		$allowed_types = array_merge( $allowed_types, [ 'image/jpg', 'image/jpe' ] );
```

Comment:
Everything about this feels counterintuitive to me. Like
- why is `'img/jpg'` in our original array, but _not_ `'img/jpe'`?
- why are we fauxing `get_allowed_mime_types()` but _not_ using the expected keys from `wp_get_mime_types()`?